ios/FluentUI/Navigation/NavigationBar.swift

```diff
@@ -62,38 +62,35 @@ open class NavigationBar: UINavigationBar {
         case system
         case custom
 
-        var tintColor: UIColor {
+        func tintColor(fluentTheme: FluentTheme) -> UIColor {
             switch self {
             case .primary, .default, .custom:
-                return Colors.Navigation.Primary.tint
+                return UIColor(dynamicColor: DynamicColor(light: fluentTheme.aliasTokens.colors[.brandForeground2].light, dark: fluentTheme.aliasTokens.colors[.foreground2].dark))
             case .system:
-                return Colors.Navigation.System.tint
+                return UIColor(dynamicColor: fluentTheme.aliasTokens.colors[.foreground2])
             }
         }
 
-        var titleColor: UIColor {
+        func titleColor(fluentTheme: FluentTheme) -> UIColor {
             switch self {
             case .primary, .default, .custom:
-                return Colors.Navigation.Primary.title
+                return UIColor(dynamicColor: DynamicColor(light: fluentTheme.aliasTokens.colors[.foregroundOnColor].light, dark: fluentTheme.aliasTokens.colors[.foreground1].dark))
             case .system:
-                return Colors.Navigation.System.title
+                return UIColor(dynamicColor: fluentTheme.aliasTokens.colors[.foreground1])
             }
         }
 
-        func backgroundColor(for window: UIWindow, customColor: UIColor?) -> UIColor {
+        func backgroundColor(fluentTheme: FluentTheme, customColor: UIColor?) -> UIColor {
+            let defaultColor = UIColor(dynamicColor: DynamicColor(light: fluentTheme.aliasTokens.colors[.brandBackground1].light, dark: fluentTheme.aliasTokens.colors[.background3].dark))
```

Comment:
Is there ever a time we actually want brandBackground1 to still be brand colored in dark mode, or should the alias token be updated to have a neutral dark mode?